fem/eltrans.cpp

```diff
@@ -245,8 +245,8 @@ int IsoparametricTransformation::TransformBack(const Vector &pt,
                                                int refinementOrder)
 {
    const int    max_iter = 16;
-   const double  ref_tol = 1e-15;
-   const double phys_tol = 1e-15*pt.Normlinf();
+   const double  ref_tol = 1e-14;
```

Comment:
I lowered the tolerance from ``1e-15`` to ``1e-14`` after finding that the algorithm could occasionally converge to a number slightly higher than ``1e-15``, causing the iteration to max out, even though it essentially found the correct point. This happened in a single case for an IntegrationPoint on the boundary of a quadratic hexahedra.

Perhaps this should be a tunable parameter, either of ``InverseTransformation`` or of the ``TransformBack`` function?